src/abstractdataframe/join.jl

```diff
@@ -347,12 +347,12 @@ end
 
 """
     innerjoin(df1, df2; on = Symbol[], makeunique = false,
-              indicator = nothing, validate = (false, false))
+              validate = (false, false))
     innerjoin(df1, df2, dfs...; on = Symbol[], makeunique = false,
               validate = (false, false))
 
-Inner join two or more data frame objects and return a `DataFrame` containing
-the result.
+Perform an inner join two or more data frame objects and return a `DataFrame` containing
```

Comment:
```suggestion
Perform an inner join of two or more data frame objects and return a `DataFrame` containing
```
Or maybe "on"? Same elsewhere.